include/onnxruntime/core/graph/graph.h

```diff
@@ -400,7 +400,7 @@ class Node {
 
   void SetNodeType(Node::Type node_type) noexcept;
 
-  void SetFunctionBody(const Function& func);
+  void SetFunctionBody(Function* func);
```

Comment:
>Function* func); [](start = 23, length = 16)

Can you explicitly set the function body to nullptr using this, or do we always expect a value? If the latter it should be a reference.